src/lib/custom-resources/cdk-guardduty-admin-setup/runtime/src/index.ts

```diff
@@ -177,16 +199,21 @@ async function isConfigurationAutoEnabled(detectorId: string, s3Protection: bool
   }
 }
 
-async function updateMemberDataSource(memberAccounts: AccountDetail[], detectorId: string, s3Protection: boolean) {
-  if (s3Protection) {
```

Comment:
It's not clear why this statement was here. I feel that this would have originally not enabled S3 protection on existing members if it was previously not enabled. Removing this check.